monkey/monkey_island/cc/resources/blackbox/clear_caches.py

```diff
@@ -32,6 +34,8 @@ def get(self, **kw):
 
         if ReportService.is_report_generated() or AttackReportService.is_report_generated():
             logger.error(NOT_ALL_REPORTS_DELETED)
+            # API Spec: Why is this 500? 500 should be returned in case an exception occurs on the
+            # server. 40x makes more sense.
```

Comment:
4xx errors mean, "you made a mistake". 5xx errors mean "the server made a mistake".  A 5xx error is appropriate here.